Sming/Makefile-rboot.mk

```diff
@@ -183,7 +183,12 @@ EXTRA_INCDIR ?= include # default to include if not set by user
 
 ENABLE_CUSTOM_LWIP ?= 1
 ifeq ($(ENABLE_CUSTOM_LWIP), 1)
-	LWIP_INCDIR = $(SMING_HOME)/third-party/esp-open-lwip/include	
+	ifeq ($(LWIP_FLAVOUR), lwip2)
+		LWIP_INCDIR = $(SMING_HOME)/compiler/include/lwip2
+		MORE_CFLAGS += -DLWIP_NO_STDINT_H=1
```

Comment:
@d-a-v Is that really needed here? Won't it be possible to add it to the lwipopts.h file or just patch the stock lwipopts.h that is coming with your repository?